.gitlab-ci.yml

```diff
@@ -10,6 +10,8 @@ stages:
 
 variables:
   GIT_SUBMODULE_STRATEGY: recursive
+  CCACHE_DIR: /cache
+  with_ccache: "true"
```

Comment:
Please add `CCACHE_MAXSIZE=100G` after moving the cache volume to the bigger partition.